### ISSUE TITLE
Cannot find File G:Sladetemp/startpage.htm

### ISSUE THREAD

**Issue by Baphomet89** [sirjuddington/SLADE#504]
**SLADE Version**: ( 3.1.1.1)
**Operating System**: Windows 7 Professional x86

**Issue Details**:

I cannot open SLADE without encountering this error. First it requests for the .htm document (which I assume is used for the homepage interface for the program) then it states, "Cannot find File G:Sladetemp/startpage.htm" and when i close it for the first time. It opens my default browser (Chrome) to a blank page and the message keeps repeating itself no matter how many times I close it, then SLADE crashes.

PC Details

Pentium Dual-core CPU @ 2.8 GHZ
RAM : 3 GB DDR3
x86 architecture
Windows 7 Professional x86

![slade3_1_0_11_what](https://cloud.githubusercontent.com/assets/18384211/14412980/0857e8c2-ff22-11e5-92e8-11cdb4e96731.JPG)

**Comment by sirjuddington** [comment]
You'll need to find your `slade3.cfg` file (should be in `%APPDATA%\SLADE3` if you type that into the address bar in file explorer). In the config file, find `temp_location_custom` and add the missing `\` to the value.

**Comment by Baphomet89** [comment]
Thanks for your help, I should have known it was somewhere in AppData that was causing this problem, but it was a FRESH install and it didn't prompt me to specify my temp directory so I assumed it was something else.

I checked the slade3.cfg file, the directory read, **"G:\SladeTemp"** for the **temp_location_custom** placeholder. I changed it to another directory and I still got the same error.

So I just deleted the slade3.cfg file and re-ran SLADE 3.1.1.1 and it works fine now.

Thanks for your help sirjuddington!
